mlflow/diviner.py

```diff
@@ -128,6 +137,14 @@ def save_model(
 
                      .. Note:: Experimental: This parameter may change or be removed in a future
                                              release without warning.
+
+    :param kwargs: Optional configurations for Spark DataFrame partition storage iff the model has
+                   been fit in Spark.
+                   Currently supported options:
+                   `partition_by` for setting a (or several) partition columns as a list of
+                   column names.
+                   `partition_count` for setting the number of part files to write from a
+                   repartition per `partition_by` group. The default part file count is 200.
```

Comment:
Nit: Can we turn this into an RST-compatible list? I think the following works:

```suggestion
    :param kwargs: Optional configurations for Spark DataFrame partition storage iff the model has
                   been fit in Spark. Currently supported options:
                   
                   - `partition_by` for setting a (or several) partition columns as a list of
                   column names.
                   
                   - `partition_count` for setting the number of part files to write from a
                   repartition per `partition_by` group. The default part file count is 200.
```

Reply:
ah very nice! TY!